ansible/roles_ocp_workloads/ocp4_workload_quarkus_super_heroes_demo/tasks/deploy_cicd.yaml

```diff
@@ -0,0 +1,270 @@
+---
+- name: Retrieve Gitea instance
+  kubernetes.core.k8s_info:
+    api_version: gpte.opentlc.com/v1
+    kind: Gitea
+    name: "{{ ocp4_workload_gitea_operator_name }}"
+    namespace: "{{ ocp4_workload_gitea_operator_project }}"
+  register: r_gitea
+
+- name: "[{{ t_project_name_argocd }}] - create project"
+  kubernetes.core.k8s:
+    state: present
+    api_version: project.openshift.io/v1
+    kind: Project
```

Comment:
I wouldn't do Project but just a namespace (with the right annotations). If you have to it would be better to use ProjectRequest instead of project. But namespace is usually enough.

Reply:
Will do.